### ISSUE TITLE
Distinguish 'chat', 'video room' and 'pre-join' better

### ISSUE THREAD

**Issue by rossng** [clowdr-app/clowdr#157]
Benjamin's original description:

> We need to better distinguish the "In chat area of room" (i.e., I'm on the "joining room" / video preview page, I can see and interact with the chat, but I can't see any video) and "Actually joined to the room" modes. One problem is that the word "Room" has multiple senses. We need to distinguish the Chat area from the Video area, both visually and terminologically. One thing that would help a lot -- and make chat rooms more useful when just chatting -- and (I hope) be quite simple to implement -- is to automatically make the video area much smaller when it's in the "Preview your camera" mode. (Ed: I agree on resizable chat as per previous points. I'm not sure I see what distinction you're making about being "in chat area" vs anything else. It feels like a non-existent distinction. BCP: The distinction is that one takes a single click to get to, while the other requires an additional click.)

Split from #76

**Comment by bcpierce00** [comment]
This is superseded by my recommendation elsewhere that the video preview
page should *not* display the room chat, and vice versa (i.e., there should
be a way to get to the room chat that does not involve video) -- these
should be entirely separate functions.

**Comment by rossng** [comment]
I think I mentioned this earlier, but my concern is that it's nice to be able to flick through rooms (without joining video) and get a sense of what's being discussed. I suspect it maybe also lowers the social barrier to joining a room with camera/mic.

We'll need to come up with a more detailed set of proposed UI changes before starting work on this.

**Comment by bcpierce00** [comment]
(1) The contents of the chat doesn't tell you much about what's being
discussed.  (2) We could have a different interface for quickly flipping
through chats.
